Add support for Pearl node

piratekev opened the pull request.

lazynina reviewed (changes requested):
@piratekev - do we need to allow connect-src to *.pearl.app? Or is image-src sufficient? If it is needed, please provide an explanation

piratekev commented:
> @piratekev - do we need to allow connect-src to *.pearl.app? Or is image-src sufficient? If it is needed, please provide an explanation

We don't need connect-src - my mistake. That's a change for running the frontend against our node.

Removed and updated. Thanks.
